less/home-page.less

```diff
@@ -1,22 +1,43 @@
 .home {
 
+  article div {
+    padding: 20px 0;
+    .divide-border();
+  }
+
   .content-section {
     margin: 30px 0;
     display: flex;
     flex-wrap: wrap;
     justify-content: space-between;
 
-   
+    @media @mobile {
+      flex-direction: column;
```

Comment:
Great job noticing you can nest the media query in the selector and use nesting on subsequent children.